.github/workflows/publish.yaml

```diff
@@ -22,30 +29,12 @@ jobs:
       - name: Publish EAS update
         uses: ./.github/actions/expo/update
         with:
-          channel: 'preview'
+          channel: '${GITHUB_REF##*/}'
```

Comment:
Why do we need to do shell string manipulations here? Isn't the ref one of (`main`, `release`, `preview`)?

Reply:
so its interesting, the ${{ github.head_ref }} works in the if statement in the yaml but appears blank when used in a command. The GITHUB_REF has the whole path of the branch (i.e. refs/heads/release), hence the manipulation